apps/qubit/modules/actor/templates/_advancedSearch.php

```diff
@@ -26,7 +26,7 @@
                 <option value="not"<?php echo 'not' == $item['operator'] ? ' selected="selected"' : ''; ?>><?php echo __('not'); ?></option>
               </select>
 
-              <input class="query" type="text" placeholder="<?php echo __('Search'); ?>" name="sq<?php echo $key; ?>" value="<?php echo $item['query']; ?>"/>
+              <input class="query" type="text" aria-label="<?php echo __('Search'); ?>" placeholder="<?php echo __('Search'); ?>" name="sq<?php echo $key; ?>" value="<?php echo $item['query']; ?>"/>
```

Comment:
Since `role="search"` is meant to be placed on the element containing all of the objects encompassing the search function, I've tried to implement this suggestion with this in mind (22942e8b633e09f681e0942652367f96bc8a02b8).

I've created landmarks on both the inline search section and entirety of advanced search section. I'm not entirely sure if this is an appropriate approach, so I'd like to know your thoughts.